CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to apimatic/core-lib-csharp
+
+Thank you for your interest in contributing! 🎉 Your contributions help make this project better. We value open-source contributions to this library, please take a few minutes to review this guide before you start.
+
+---
+
+## 💡 How to Contribute
+
+### 📌 1. Reporting Issues
+
+- Search existing issues before opening a new one.
+- Use a **descriptive title** and provide clear steps to reproduce.
+- Include relevant logs, screenshots, or error messages.
+
+### 🔧 2. Making Changes
+
+#### Step 1: Create a Branch
+
+- Create a new branch from `master`:
+  ```sh
+  git checkout -b your-feature-name
+  ```
+
+#### Step 2: Make Your Changes
+
+- Follow the project's **coding standards**.
+asadali214 marked this conversation as resolved.
```

Comment:
I think you have accidently copied resolved conversation lines as well, please remove all such lines